src/figuresClasses.ts

```diff
@@ -1,19 +1,70 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string = 'triangle';
```

Comment:
shouldn't be a `string` type. you have to create the corresponding enum for the shape